example/README.md

```diff
@@ -1,27 +1,15 @@
-# Example app
+# Yoti IDV, Example
 
-This file contains installation instructions for the example app.
-The instructions assume you have a [working React Native development environment](https://reactnative.dev/docs/getting-started) for Android and/or iOS.
+An example of how to run the react native wrapper of Yoti IDV for [Android](https://github.com/getyoti/yoti-doc-scan-android) and [iOS](https://github.com/getyoti/yoti-doc-scan-ios).
 
-## Installation
-- Navigate to the example directory: `cd example`
-- Install dependencies: `yarn install`
-- Install pods if you intend to run on iOS: `cd ios && pod install`
-
-# Running on Android (via Android Studio)
+## Prerequisites
+To run this example, a working working React Native environment is needed (see [reactnative.dev/docs/getting-started](https://reactnative.dev/docs/getting-started)).
 
-- In the example folder, run: `yarn start`
-- Open the `example/android` project in Android Studio
-- Sync Gradle
-- Run a build as you regularly would, preferrably on a device with the Yoti app installed
-- If you are running on a physical device, remember to execute `adb -s DEVICE_ID_HERE reverse tcp:8081 tcp:8081`. You can find the appropriate device id by running `adb devices`
-
-# Android (using React Native CLI)
-- Simply execute `yarn run android` in the `example` folder
```

Comment:
This section is useful, it's way easier to simply run the app like this, I would not remove it.

Reply:
wasn't removed. just moved.